portal/src/pages/components/accordion.tsx

```diff
@@ -1,32 +1,26 @@
-// @ts-ignore
-import ButtonExampleCode from "!raw-loader!../../examples/AccordionExample";
-// @ts-ignore
-import AccordionMarkdown from "!raw-loader!@fremtind/jkl-accordion/README.md";
 import React from "react";
 import ReactMarkdown from "react-markdown";
-import { AccordionExample } from "../../examples/AccordionExample";
+
 import Layout from "../../components/Layout";
-import { PrismHighlightedCode } from "../../components/PrismHighlightedCode";
+import { Example } from "../../components/Example";
 import { renderer } from "../../presentation/markdownRenderer";
+
+/* Import readme fil with !raw-loader! */
+// @ts-ignore
+import readmeContents from "!raw-loader!@fremtind/jkl-accordion/README.md";
+/* Import code to be rendered as example */
+import { accordionExample } from "../../examples/AccordionExample";
+/* Import components used in the example, and expose them in an object */
 import { Accordion, AccordionItem } from "@fremtind/jkl-accordion-react";
+const components = { Accordion, AccordionItem };
 
 export default function Button() {
     return (
         <Layout>
-            <h2 className="jkl-h1 portal-content__title">Trekkspillliste</h2>
-            <div>
-                <div className="portal-content__example">
-                    <AccordionExample />
-                </div>
-
-                <Accordion>
-                    <AccordionItem title="Kode">
-                        <PrismHighlightedCode>{ButtonExampleCode}</PrismHighlightedCode>
-                    </AccordionItem>
-                </Accordion>
-            </div>
+            <h2 className="jkl-h1 portal-content__title">Trekkspilliste</h2>
```

Comment:
Noen grunn til at vi ikke dogfooder og bruker <H2>?

Reply:
Mest for å slippe all importeringa.. Er jo forsåvidt fortsatt dogfooding all den tid vi bruker klassene fra stilpakka ;)